### ISSUE TITLE
Typo in Machine Learning Explore Card - Supervised vs. Unsupervised Subchapter

### ISSUE THREAD

**Issue by JonMrowczynski** [LeetCode-Feedback/LeetCode-Feedback#17051]
#### Your LeetCode username
jonmrowczynski

#### Category of the bug
- [ ] Question
- [ ] Solution
- [x] Language
- [ ] Missing Test Cases 

#### Description of the bug
In the Machine Learning Explore Card - Chapter "Machine Learning - What", Subchapter "Supervised vs. Unsupervised", There is a typo as indicated in the attached screenshot. 

#### Code you used for Submit/Run operation
N/A

#### Language used for code
N/A

#### Expected behavior
Basically "extend" should be "extent".

#### Screenshots
![Machine Learning Typo 2](https://github.com/LeetCode-Feedback/LeetCode-Feedback/assets/39063205/eee12255-94fe-4195-b779-0cb0038804b6)

#### Additional context
N/A

**Comment by LC-Pam** [comment]
Hi there,

Thank you for reaching out to us.

 We've relayed this issue to our team to investigate. 

We will reach back to you after we have the results.

Thank you for your support.

**Comment by LC-Pam** [comment]
Thank you for your time. 

We've used your feedback to update the problem. 

Your LeetCode account has received 100 LeetCoins as a reward for this feedback. 

If you have any other questions or feedback, please don't hesitate to let us know! 

We appreciate your support!
